package.json

```diff
@@ -119,7 +119,7 @@
 		"@types/react-router-dom": "^4.3.3",
 		"@types/react-select": "^2.0.17",
 		"@types/react-virtualized": "^9.21.1",
-		"@types/styled-components": "^4.1.15",
+		"@types/styled-components": "4.1.8",
```

Comment:
Why the pinning?

Reply:
It fixes the issue with `@types/react-native`, see issue99

Reply:
Gotcha. Would the postinstall rm for the bad types solution be better for us?

Reply:
I don't see any disadvantage of just pinning version, everything still works

Reply:
Sounds good